src/main/java/seedu/multitasky/model/ModelManager.java

```diff
@@ -158,6 +158,15 @@ public void updateEntry(ReadOnlyEntry target, ReadOnlyEntry editedEntry)
     }
     // @@author
 
+    @Override
+    public void undoPreviousAction() throws NothingToUndoException {
+        indicateUndoAction();
+    }
+
+    @Override
+    public void redoPreviousAction() throws NothingToRedoException {
+        indicateRedoAction();
+    }
     // =========== Filtered Entry List Accessors ===========
```

Comment:
@akd13 
I moved around methods in `ModelManager.java` quite a bit in my PR and grouped all storage-related methods in one section. I'll merge my branch into master for a while so that you can merge master into your branch and edit from there. Helps to avoid some merge conflicts